runtime/doc/api.txt

```diff
@@ -232,7 +232,45 @@ An example of calling the api from vimscript: >
 
     " later
     call nvim_buf_clear_namespace(0, src, 0, -1)
+
+
+==============================================================================
+Floating windows						   *api-floats*
```

Comment:
Perhaps `api-floatwin` is a better help tag, to avoid overlap with "floating point numbers".